teuthology/suite/run.py

```diff
@@ -516,13 +520,45 @@ def schedule_suite(self):
         ).name
         self.base_yaml_paths.insert(0, base_yaml_path)
 
+        # compute job limit in respect of --sleep-before-teardown
+        job_limit = self.args.limit or 0
+        sleep_before_teardown = int(self.args.sleep_before_teardown or 0)
+        if sleep_before_teardown:
+            if job_limit == 0:
+                log.warning('The --sleep-before-teardown is provided, '
+                            'only 1 job will be scheduled. '
+                            'Use --limit to run more jobs')
+                # give user a moment to read this warning
+                time.sleep(5)
+                job_limit = 1
+            elif self.args.non_interactive:
+                log.warning(
+                    'The --sleep-before-teardown is used and there will be maximum '
```

Comment:
Could it please be `The --sleep-before-teardown option is active. There will be a maximum of {} jobs . . .`

i.e.,

* insert "option"
* s/used/active/
* s/and there/. There/
* insert "a" before "maximum"

Reply:
As you wish